src/CommunityToolkit/Xamarin.CommunityToolkit/Views/BadgeView/BadgeView.shared.cs

```diff
@@ -157,6 +159,25 @@ public FontAttributes FontAttributes
             set => SetValue(FontAttributesProperty, value);
         }
 
+        static void IsRoundedChanged(BindableObject bindable, object oldvalue, object newvalue) =>
```

Comment:
I suggest to follow the same order creation of BindableProperty, getter/setter, property changed event:
````
BindableProperty.Create
...
public bool IsRounded
...
static void IsRoundedChanged(BindableObject bindable, object oldvalue, object newvalue) => 
```

Reply:
Changed!